cassandraprotocol/frame/rawframe.go

```diff
@@ -0,0 +1,14 @@
+package frame
+
+import (
+	"fmt"
+)
+
+type RawFrame struct {
+	Header *RawHeader
+	Body   []byte
```

Comment:
maybe `Body io.Reader` so that we can pass in other structures than just `[]byte`?

Reply:
I don't see the point in doing that, do you have a specific use case in mind?

Reply:
It's how it's going to be consumed by cloud-gate:

https://github.com/riptano/cloud-gate/blob/b61d1985fd2137637d9b0811978ad9a9a0e507aa/proxy/pkg/cloudgateproxy/cqlparser.go#L66-L86

Reader being an interface, this enables consumers of `RawFrame` to use whatever implementation they have.

Reply:
You can create a reader with `bytes.Reader` anyway and if you don't provide a `[]byte` then anytime the caller needs to read the body in multiple places of the code, they will have to store that `[]byte` anywhere or they will be allocating multiple `[]byte` for the same body, no?